Cargo.toml

```diff
@@ -5,7 +5,7 @@ prettytable-rs = "^0.10"
 rand = "^0.8"
 
 [dependencies.arrow2]
-features = ["compute"]
+features = ["compute", "io_ipc"]
```

Comment:
Required for the IPC roundtrip workaround for the struct/fixed-size list array bug(s) on the arrow2 --> pyarrow FFI boundary.